doc/whats_new.rst

```diff
@@ -78,6 +78,16 @@ Changelog
 New features
 ............
 
+   - :class:`ensemble.GradientBoostingClassifier` and
+     :class:`ensemble.GradientBoostingRegressor` now support early stopping
+     via `n_iter_no_change`, `validation_fraction` and `tol`. :issue:`7071` by
+     `Raghav RV`_
+
+   - :class:`model_selection.GridSearchCV` and
+     :class:`model_selection.RandomizedSearchCV` now support simultaneous
+     evaluation of multiple metrics. Refer to the
+     :ref:`multimetric_grid_search` section of the user guide for more
+     information. :issue:`7388` by `Raghav RV`_
 Classifiers and regressors
```

Comment:
I don't get you? This is indented the same way all other entries are. Am I missing something?